service/dap/server.go

```diff
@@ -1555,6 +1563,30 @@ func (s *Server) onCancelRequest(request *dap.CancelRequest) {
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
+// onExceptionInfoRequest handles 'exceptionInfo' requests.
+// Capability 'supportsExceptionInfoRequest' is set 'initialize' response.
+func (s *Server) onExceptionInfoRequest(request *dap.ExceptionInfoRequest) {
+	goroutineID := request.Arguments.ThreadId
+	body := s.exceptions[goroutineID]
+	frames, err := s.debugger.Stacktrace(goroutineID, s.args.stackTraceDepth, 0)
+	if err == nil && len(frames) > 0 {
+		apiFrames, err := s.debugger.ConvertStacktrace(frames, nil)
+		if err == nil {
+			var buf bytes.Buffer
+			terminal.PrintStack(func(path string) string {
+				return s.toClientPath(path)
+			}, &buf, apiFrames, "\t", false)
+
+			body.Details.StackTrace = "Stack:\n" + buf.String()
```

Comment:
Do `fmt.Fprintf(&buf, "Stack:\n")` before the call to PrintStack instead of concatenating here.

Reply:
Done.